server/migrations/2023-02-14-193708_initial_schema/up.sql

```diff
@@ -0,0 +1,10 @@
+-- Your SQL goes here
+
+CREATE TABLE mina_proposals (
+  id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+  key TEXT NOT NULL,
```

Comment:
title seems more accurate than key. I would also add a version field but it could be a different task

Reply:
I understand "key" is the value from the URL for backwards compatibility